code/API_definitions/number-verification.yaml

```
@@ -2,43 +2,67 @@ openapi: 3.0.3
+    Number Verification API performs real-time checks to verify the phone number of the mobile device being used to access a service provider (SP) service.
 
+    The mobile phone number is personal data and end-user consent might be required.
 
     # Relevant Definitions and Concepts
 
-    - **Network-Based Authentication**: Authentication mechanism based on the identification of the endpoint of a network connection. Network operators know to which subscriber a network resource is assigned at a given moment, for example the mobile phone number associated to a specific mobile network connection.
-
+    - **Network-Based Authentication**: Authentication mechanism based on the identification of the endpoint of a network connection.
+    Network operators know to which subscriber a network resource is assigned at a given moment, for example the mobile phone number associated to a specific mobile network connection.
+    - **SIM-Based Authentication**: Authentication mechanism based on the identification of the subscriber's SIM installed in the user's device.
     # API Functionality
 
-    This enables a Service Provider (SP) to verify the phone number of the mobile device being used to access their service where the mobile device is accessing the *service provider* over a mobile network (WiFi connections are out of this API scope). This can happen either by getting the comparison result or receiving the phone number of the device that is used, so they can verify it themselves.
+    This API enables an API Consumer to verify or retrieve the phone number of the mobile device being used to access their service
+
+    # The Authentication Request
+
+    ## Authentication Request with a temporary token
+
+    If the API Consumer has a TS.43 temporary token created on the mobile device then this API works over all connections e.g. WiFi taking advantage of the SIM-Based authentication.
+    How the API Consumer gets a TS.43 temporary token is out-of-scope of the API definition.
+    The API Consumer sends the temporary token to their backend and which sends a CIBA Authentication Request, as described in the current release [CAMARA APIs Access and User Consent Management](https://github.com/camaraproject/IdentityAndConsentManagement), with a parameter "login_hint=operatortoken:<temporary token>".
+
+    ## Authentication Request without a temporary token
+
+    If the API Consumer does not have a TS.43 temporary token then the API Consumer must use OpenId Connect Authorization Code Flow as described in the current release of [CAMARA APIs Access and User Consent Management](https://github.com/camaraproject/IdentityAndConsentManagement).
+    This mechanism of utilizing the mobile connection does not work on a fixed-net Internet connection nor over WiFi or VPN.
```

I think it is better just to say the device must be connected to the mobile network if the API consumer does not have a TS.43 token.
```suggestion
    For this method of authentication to work, the device must be connected to the mobile network.
```